Sources/Core/Internal/CGImage+Load.swift

```diff
@@ -0,0 +1,36 @@
+/**
+ *  Imagine Engine
+ *  Copyright (c) John Sundell 2017
+ *  See LICENSE file for license
+ */
+
+import Foundation
+import CoreGraphics
+
+internal extension CGImage {
```

Comment:
Might be missing something, but how come this can't be part of `BundleTextureImageLoader`? Loading images seems to be its job 😄